### PR TITLE
Add an update function for a progress bar

### PR DESCRIPTION
**Pull request by ShcherbaDev**
Solution for this [ticket](https://phabricator.wikimedia.org/T232457) from Phabricator. Works with [these](https://github.com/gopavasanth/video-cut-tool/pull/24) changes in [video-cut-tool](https://github.com/gopavasanth/video-cut-tool) repository.
Used library "[socket.io](https://www.npmjs.com/package/socket.io)".

### CODE REVIEW

**Comment by gopavasanth** [comment]
Hi @ShcherbaDev did you pull the latest code and made this commit?

**Comment by ShcherbaDev** [comment]
> Hi @ShcherbaDev did you pull the latest code and made this commit?

I have problems with that

**Comment by gopavasanth** [comment]
I hope you will resolve the code conflicts soon and please squash all your commits and make a final commit :)

File: .gitkeep [hunk 1]
```diff
@@ -1,2 +1,5 @@
 #videos
 routes/videos
+
+#public
+public/
```
Comment (gopavasanth) [inline .gitkeep]: I already fixed this at https://github.com/gopavasanth/video-cut-tool-back-end/commit/3c452dd142bae7fa4e2d5c8671c1f3f1c8d2dd4f

Comment (ShcherbaDev) [reply .gitkeep]: I have merge conflicts that I can't resolve normally.

Comment (gopavasanth) [reply .gitkeep]: Aah, it's little time taking to resolve such conflicts but I'm sure you will do it sooner :)

Comment (ShcherbaDev) [reply .gitkeep]: Yep, finally resolved

**Pull request merged**